conda_devenv/devenv.py

```diff
@@ -516,6 +568,13 @@ def main(args=None):
     parser.add_argument(
         "--quiet", action="store_true", default=False, help="Do not show progress"
     )
+    parser.add_argument(
+        "--env_var",
```

Comment:
```suggestion
        "--env-var",
```
Is more common for cli options.